index.js

```diff
@@ -1,17 +1,52 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
```

Comment:
jejeje secretillo, el metodo push devuelve el length del array una vez hecho el push asi que puedes hacer esto!
```suggestion
    this.length = this.items.push(item);
    this.items.sort((a, b) => a - b);
```

Reply:
Es buenisimo!!! muchas gracias!!!